superbench/runner/runner.py

```diff
@@ -78,7 +78,7 @@ def __validate_sb_config(self):    # noqa: C901
                 elif mode.name == 'torch.distributed':
                     if not mode.proc_num:
                         self._sb_benchmarks[name].modes[idx].proc_num = 8
-                elif mode.name == 'mpi':
+                elif 'mpi' in mode.name:
```

Comment:
Please keep the exact match.

`elif mode.name == 'mpi' or mode.name == 'mpi-parallels'`